src/ui/FieldImage/ImageFieldValue.ts

```diff
@@ -0,0 +1,89 @@
+import { Component, Initialization } from '../../Core';
+import { exportGlobally } from '../../GlobalExports';
+import { IComponentBindings } from '../Base/ComponentBindings';
+import { IQueryResult } from '../../rest/QueryResult';
+import { FieldValue, IFieldValueOptions } from '../FieldValue/FieldValue';
+import { ComponentOptions, IFieldOption } from '../Base/ComponentOptions';
+
+export interface IImageFieldValue {
+  field?: IFieldOption;
+  width?: string;
+  height?: string;
+}
+
+/**
+ * The ImageFieldValue component displays the value of a field associated to its parent search result.
+ * It is use strictly for displaying an image.
+ *
+ * This component is a result template component (see [Result Templates](https://developers.coveo.com/x/aIGfAQ)).
+ *
+ * A common use of this component is to display the image of a product for sale.
+ */
+export class ImageFieldValue extends Component {
+  static ID = 'ImageFieldValue';
+  static doExport = () => {
+    exportGlobally({
+      ImageFieldValue: ImageFieldValue
+    });
+  };
+
+  /**
+   * The options for the component
+   * @componentOptions
+   */
+  static options: IImageFieldValue = {
+    /**
+     * Specifies the field that the ImageFieldValue should display.
+     *
+     * Specifying a value for this parameter is required in order for the ImageFieldValue component to work.
+     *
+     * **Default value:** `@field`
+     */
+    field: ComponentOptions.buildFieldOption({ defaultValue: '@field', required: true }),
+    /**
+     * The width of the image
+     *
+     * the value can either be in `pixels` or `pourcentages`
```

Comment:
Are we sure we want to support  width / height values expressed as percentages?

Unless I'm mistaken, only pixel values are supported for those attributes in HTML5.

Reply:
So I tried it in a lot of different browsers and it seems to work but, you are right. [HTML5 only support pixels in those options.](http://w3c.github.io/html-reference/img.html)

To avoid any unforeseen problem, I will change to only accepting pixels